addons/mass_mailing/tests/test_mass_mailing_controllers.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import HttpCase
+
+
+class TestMassMailingControllers(HttpCase):
```

Comment:
Would rename the file to test_mailing_controllers.py (other files will be renamed soon :) )